documentation/src/main/asciidoc/user_guide/configuration.adoc

```diff
@@ -252,6 +253,22 @@ GlobalConfiguration globalConfig = new GlobalConfigurationBuilder().transport()
 
 ----
 
+===== Using a custom JChannel
+If you want to construct the JGroups link:http://www.jgroups.org/manual4/index.html#JChannel[JChannel] by yourself, you can do so.
+
+NOTE: The JChannel must not be already connected.
+
+[source,java]
+----
+GlobalConfigurationBuilder global = new GlobalConfigurationBuilder();
+JChannel jchannel = new JChannel();
+// Configure the jchannel to your needs.
+JGroupsTransport transport = new JGroupsTransport(jchannel);
```

Comment:
we have a JGroupsChannelLookup, but I guess it isn't easier to use when compared with your example :) https://github.com/infinispan/infinispan/blob/master/core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsChannelLookup.java